impeller/renderer/formats.h

```diff
@@ -87,7 +87,10 @@ enum class PixelFormat {
   kR8G8B8A8UNormIntSRGB,
   kB8G8R8A8UNormInt,
   kB8G8R8A8UNormIntSRGB,
+
+  // Depth and stencil formats.
   kS8UInt,
+  kD32FloatS8UInt,
```

Comment:
I never noticed either. For the depth in impeller-cmakeI literally used [B8G8R8A8](https://github.com/bdero/impeller-cmake-example/blob/ff71cda2136128bc65017c3b62bcadd5f4a6d41d/src/main_gles.cc#L234) because it's GLES only right now and the attachments have no affect on the surface framebuffer. 😄